docs/CONTRIBUTING.md

```diff
@@ -104,3 +105,29 @@ Development standards will be rigorously enforced on this project. Drupal mainta
 - [Twig coding standards](https://www.drupal.org/docs/develop/coding-standards/twig-coding-standards)
 - [JavaScript coding stand](https://www.drupal.org/docs/develop/standards/javascript)
 - [CSS Architecture](https://www.drupal.org/docs/develop/standards/css/css-architecture-for-drupal-8)
+
+### Linting
+
+To run the linting, first you'll need to gain access to the [yalesites-org/eslint-config-and-other-formatting](https://github.com/yalesites-org/eslint-config-and-other-formatting) repo if you don't already have access.  
```

Comment:
This isn't actually necessary. It's a public repo, and it's released as a public package. So the only requirement is that they have a personal access token saved. That's a requirement of GitHub for installing packages, both private and public.

Reply:
Removed.  Thanks.